### ISSUE TITLE
Images are extremely blurry

### ISSUE THREAD

**Issue by adamreisnz** [brave/brave-browser#5405]
## Description 
Images in Brave are extremely blurry in some cases.

## Steps to Reproduce
Compare this Github page in Firefox (left) and Brave (right) and look at the avatars:

![image](https://user-images.githubusercontent.com/490562/61944910-e3c81d80-aff2-11e9-85c1-2ff4d21dfbc7.png)

## Actual result:
Avatar is so blurry I almost don't recognise myself.

## Expected result:
For the avatar to be sharp, as I uploaded it and as other browsers display it.

## Reproduces how often: 
100%

## Brave version (brave://version info)

Brave | 0.66.100 Chromium: 75.0.3770.142 (Official Build) (64-bit)
-- | --
Revision | 8d93477bd18154c67a70a67019cb8032382c55dd-refs/branch-heads/3770@{#1160}
OS | Mac OS X

## Version/Channel Information:
Chrome also seems to be affected.

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields?
No
- Does the issue resolve itself when disabling Brave Rewards?
No
- Is the issue reproducible on the latest version of Chrome? 
Yes

## Miscellaneous Information:
I note that when creating a new issue, the avatar there looks sharp (both screenshots from Brave):

![image](https://user-images.githubusercontent.com/490562/61945189-7c5e9d80-aff3-11e9-91ff-ceb541446723.png)

Upon inspecting the code, it seems the difference is that the blurry image is originally larger. So it seems the algorithm to downsize images is broken, as images which are larger should, when downsize, be sharper not blurrier.

**Comment by kjozwiak** [comment]
@adamreisnz can you try and see if you get the same results on Chrome? I went through a few images on GH and it seems like Firefox renders them more clearly compared to Chromium browsers. Would be interesting to see if Chrome/Opera and other Chromium based browsers are having the same issue compared to Firefox.

**Comment by adamreisnz** [comment]
Yes, Chrome has the same problem as I mentioned also in the original issue.

I don't think Firefox renders them more clearly necessarily, because Brave does render the image that is downsized from 88px to 44px with the same clarity.

It's when Brave/Chrome scale the 180px image down to 44px when the algorithms seem to fail and render blurry images instead.

I've noticed this started to happen a few months ago.

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
